networkit/test/test_clique.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+import unittest
+import os
+import networkit as nk
+import numpy as np
+import scipy
+
+class TestCliques(unittest.TestCase):
+
+	
+	def testMaxClique(self):
+		G = nk.readGraph("input/jazz2_directed.gml",nk.Format.GML)
+		G.removeSelfLoops()
+		#without callback, testing maxOnly, expecting single Clique
+		A1 = nk.clique.MaximalCliques(G, maximumOnly=True).run().getCliques()
+		self.assertEqual(len(A1), 1)
+		#with callback
+		def testCallback():
+			return [1,2,3]
```

Comment:
I don't think this is how the callback is supposed to be used. Please check the `MaximalClique` C++ tests for an example -- I am not suggesting to test the algorithm twice, just make sure that the callback works correctly also in Python.